app/MMB/src/navigation/TravelDocumentModalScreen.js

```diff
@@ -158,4 +229,13 @@ const styles = StyleSheet.create({
   datePickerContainer: {
     flex: 1,
   },
+  error: {
+    alignSelf: 'flex-end',
+  },
+  noExpiry: {
+    color: '#DFE4EA',
```

Comment:
Shall we add this color to the `Color.js`?

Reply:
🤷‍♂️ Colors are currently a mess from the designers from my POV. 

This color is for now only used one place. If it should be reused somehow, we find a name for it and add it to `Color.js`

Reply:
I disagree a bit: if you are using a new color, then you must be diligent enough to check if it already exists in the codebase, add it to the `Color.js` file and then reflect changes to the other which has probably nothing to do with your current task.
As opposed to just checking for colors in `Color.js`.

Reply:
If we search for `'#` in our project, we have 77 results in 22 files. 

Probably room for improvements, because a lot of them are just white and black, and color codes I think we already have in colors. 

But ok, I will move this color to the file 😖

Reply:
I will do this cleaning right now then.